website/docs/src/scss/_code.scss

```diff
@@ -0,0 +1,118 @@
+/* CODE */
+
+.codeBlockContainer__-_-node_modules-\@docusaurus-theme-classic-lib-next-theme-CodeBlock-styles-module {
+  border: solid 1px var(--ifm-toc-border-color);
+}
+
+code {
+  border: none !important;
+  background: transparent !important;
+}
+
+.navbar {
+  border-bottom: 1px solid var(--ifm-toc-border-color);
+}
+
+.token {
```

Comment:
What about creating our own syntax-highlight theme? This would avoid all those `important`: https://docusaurus.io/docs/markdown-features/code-blocks#theming